### PR TITLE
phone number exercise

### PR DESCRIPTION
**Pull request by gabrielperales**
~~Just to let you know that I'm working on it.~~

### CODE REVIEW

File: exercises/phone-number/__tests__/PhoneNumber_test.re [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  test("cleans numbers with dots", () =>
+    expect(phoneNumber("223.456.7890")) |> toEqual(Some("2234567890"))
+  );
+  test("cleans numbers with multiple spaces", () =>
+    expect(phoneNumber("223 456   7890   ")) |> toEqual(Some("2234567890"))
+  );
+  test("invalid when 9 digits", () =>
+    expect(phoneNumber("123456789")) |> toEqual(None)
+  );
+  test("invalid when 11 digits does not start with a 1<Paste>", () =>
```
Comment (tejasbubane) [inline exercises/phone-number/__tests__/PhoneNumber_test.re]: remove `<Paste>`

File: exercises/phone-number/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Basic Reason Template
+
+Hello! This project allows you to quickly get started with Reason and BuckleScript. If you wanted a more sophisticated version, try the `react` template (`bsb -theme react -init .`).
+
```
Comment (tejasbubane) [inline exercises/phone-number/README.md]: please add the problem statement to this readme. https://github.com/exercism/problem-specifications/blob/master/exercises/phone-number/description.md

**Comment by tejasbubane** [comment]
Hi @gabrielperales thanks for the contribution. We recently configured travis CI in this repo. Can you rebase your branch on the updated master so that that the tests can run? 

Feel free to comment if any issues.

File: exercises/phone-number/__tests__/PhoneNumber_test.re (hunk 1 above)
Comment (gabrielperales) [reply exercises/phone-number/__tests__/PhoneNumber_test.re]: @tejasbubane Thanks, sometimes VIM makes weird stuff...

File: exercises/phone-number/README.md (hunk 2 above)
Comment (gabrielperales) [reply exercises/phone-number/README.md]: @tejasbubane yep, it still being a _work in progress_, it is not ready to be merged or reviewed but thanks anyway.

**Comment by gabrielperales** [comment]
@tejasbubane fixed, updated, and ready to review ;)

File: exercises/phone-number/.gitignore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.DS_Store
```
Comment (stevejb71) [inline exercises/phone-number/.gitignore]: There's a top level gitignore, so this is not needed.

**Comment by tejasbubane** [comment]
@stevejb71 I will add the interface file for this one along with other exercises. Apart from that this one looks good to go.

**Pull request merged**